src/sentry/integrations/bitbucket_server/webhook.py

```diff
@@ -67,7 +67,7 @@ def __call__(self, organization, integration_id, event):
             from_hash = None if change.get("fromHash") == "0" * 40 else change.get("fromHash")
             for commit in client.get_commits(
                 project_name, repo_name, from_hash, change.get("toHash")
-            )["values"]:
```

Comment:
@marbon87 Can you add a unit test to catch this issue?

Reply:
A unit doesn't make much sence because, does it? We need to test the contract between webhook and client.

Sadly i am failing in bootstraping a test by now.

Reply:
@marbon87 Maybe I am using the term "unit test" loosely here. But in Sentry, we usually would not test the contract between the webhook in client because we don't usually have unit tests for the client. Instead, we generally test the application code and mock the requests which would check the functionality of both the webhook and the client. That being said, I see we don't even have any tests for the bitbucket server webhooks so it would be asking a lot of you to start that. I will approve the PR and merge it on Monday.

Reply:
Hi, can you merge the pr please

Reply:
@marbon87 Thanks for the reminder. I forgot about it over the long weekend :D

Reply:
Thanks a lot!